documentation/operators/src/testing/docker-monitor.md

```diff
@@ -0,0 +1,246 @@
+<!--
+TODO
+1. Add a few points of advantages why to run it in this way (clean, remote from environment -> safer,  can use dockerhub etc etc)
+2. Explain in Prometheus setup (in config step) how to get the metrics key and how to add it to the node config (that will be documented in node-setup.md)
+3. Add grafana setup
+4. Add prometheus setup
+5. Add node exporter setup
+6. Add how to view it in the browser (possible to use the setup from explorenym guide
+7. Additionally add how to spin up an ansible template
+-->
+
+# Setup Prometheus and Grafana in a Docker
+
+This setup may seem more complex than running a few scripts, but it has certain advantages.
+- Running things in a container
+<!-- ADD DVANTEGES OF DOCKER SETUP -->
+
+## Docker, Prometheus and Grafana Metric SETUP
+
+This setup can be done from the same VPS like your node or on a completelly remote machine. In any case it's setup in a Docker container so it works remotely from the environment where your node runs.
+
+### Docker setup
+
+<!--INIT AND START DOCKER CONTAINER -->
+
+### Prometheus Setup
+
+This entire installation shall be done with `root` privileges. If you not `root`, start with `su` command before the following steps.
```

Comment:
'This entire installation shall be done with `root` privileges.' -> 'This installation requires root privileges.'

Reply:
So people have to run _as_ root for this, they can't just `sudo` the commands?